### DISCUSSION TITLE
Greek Language

### DISCUSSION THREAD

**Discussion by idpetrocheilos** [PSAppDeployToolkit/PSAppDeployToolkit#689] [General]
Hello, due to my business needs I translated the UI into Greek language. Is there anyway that I can post this so that it becomes available for others too? People may not have the time to translate everything. I am new to GitHub so sorry if this question is lame. Thank you for your time!

**Comment by That-Annoying-Guy** [comment]
Thank you very much.
I'm sure the developers will be keen to add it in.

I would try to post it here:
https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/issues

Try to include the **Greek** section of your AppDeployToolkitConfig.xml file in the post.
And mention if you are interested to be a translator for the Greek language.

If you need help, I help out when I can at:
https://discourse.psappdeploytoolkit.com/c/thetoolkit/general-discussion

**Comment by mjr4077au** [comment]
This has been added (and re-added).
